src/renderer/components/QueryHeader/QueryHeader.css

```diff
@@ -1,22 +1,11 @@
 .QueryHeader {
   position: relative;
-}
-
-.QueryHeader-inputTitle {
-  font-weight: bold;
-  font-size: 24px;
-  margin: 0;
-  padding: 10px;
-  outline: none;
-  border: none;
-  font-family: inherit;
-  width: 100%;
-  height: 28px;
+  display: flex;
+  flex-direction: row;
+  justify-items: space-between;
 }
 
 .QueryHeader-selectDataSource {
-  position: absolute !important;
-  right: 10px;
-  top: 6px;
+  margin: auto 10px auto 20px;
```

Comment:
Use flexbox over position-absolute not to overlap between query header title and select element.